src/Admin/JecouteLocalSurveyAdmin.php

```diff
@@ -127,7 +134,7 @@ protected function configureListFields(ListMapper $listMapper)
             ->add('getQuestionsCount', null, [
                 'label' => 'Nombre de questions',
             ])
-            ->add('author.referentTags', null, [
+            ->add('getTagsAsString', null, [
```

Comment:
could you try to use `array` type without `getTagsAsString`, ex: 
```
add('tags', 'array')
```

Reply:
Nice idea :+1: